package.json

```diff
@@ -11,7 +11,9 @@
     "test": "jest",
     "testWithCoverage": "jest --coverage",
     "lint": "eslint src tests --ext .ts",
-    "lint:fix": "eslint src tests --ext .ts --fix"
+    "lint:fix": "eslint src tests --ext .ts --fix",
+    "prepublish": "rm -rf dist/ && npm run-script build",
```

Comment:
NIT: `rm -rf` is not available on windows.
[rimraf](https://www.npmjs.com/package/rimraf) or something is cross-platform `rm` command.